webapp/components/test-run.html

```diff
@@ -104,6 +116,9 @@
         }
 
         return match[1];
+
```

Comment:
Looks like we missed a `}` while merging `master` back into this feature branch. I'll rebase and resolve this